style.css

```diff
@@ -1,340 +1,1391 @@
+}
+
+.logo__image {
+    margin-right: 15px;
+}
+
+.logo__text {
+    font-family: var(--font-secondary);
+    font-size: 32px;
+    line-height: 42px;
+    text-align: left;
+    color: #ffffff;
+    text-decoration: none;
+    text-decoration-style: none;
+    text-decoration-line: none;
+    text-transform: uppercase;
+}
+
+.social {
+    display: flex;
+    column-gap: 10px;
+}
+
+.social__item {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 40px;
+    height: 40px;
+}
+
+.social__item a {
+    display: inherit;
+}
+
+.social__item--small {
+    width: 32px;
+    height: 32px;
+    border: 2px solid #D0D7FB2B;
+}
+
+/* Index page FAQs block */
+
+.faq {
+    width: 960px;
+    margin: 0 auto 40px;
+}
+
+.faq__item {
+    position: relative;
+    margin-bottom: 15px;
+    padding: 24px;
+    background-color: rgba(29, 28, 40, 1);
+}
+
+.faq__question {
+    font-family: var(--font-secondary);
+    font-size: 24px;
+    line-height: 32px;
+    text-underline-position: from-font;
+    text-decoration-skip-ink: none;
+}
+
+.faq__answer {
+    margin-top: 20px;
+    padding-top: 20px;
+    border-top: 1px solid rgba(191, 197, 238, 0.12);
+}
+
+details>summary {
+    list-style: none;
+}
+
+summary::after {
+    position: absolute;
+    content: url('images/plus.svg');
+    right: 24px;
+}
+
+details[open] summary:after {
+    position: absolute;
+    content: url('images/minus.svg');
+    right: 24px;
+}
+
+/* Footer */
+
+.footer {
+    background-color: #1A1925;
+}
+
+.footer__main {
+    display: flex;
+    justify-content: space-between;
+    max-width: 1630px;
+    padding: 60px 15px 40px 15px;
+    margin: 0 auto;
+    border-bottom: 1px solid #BFC5EE1F;
+}
+
+.footer .logo__text {
+    color: var(--color-blue);
+}
+
+.footer-nav {
+    display: flex;
+}
+
+.footer-nav__block {
+    margin-left: 20px;
+}
+
+.footer-nav__title {
+    display: block;
+    padding-bottom: 10px;
+    font-size: 20px;
+    font-weight: 500;
+    line-height: 28px;
+}
+
+.footer-nav__item {
+    padding: 2px 30px 2px 0;
+}
+
+.footer__description {
+    max-width: 320px;
+    margin-top: 10px;
+    font-size: 14px;
+    line-height: 20px;
+}
+
+.footer__secondary {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    max-width: 1630px;
+    margin: 0 auto;
+    padding: 10px 15px;
+}
+
+.footer__copyright {
+    font-size: 14px;
+    line-height: 20px;
+}
+
+.footer__copyright span {
+    text-transform: uppercase;
+}
+
+.footer__social img {
+    vertical-align: top;
+}
+
+.catalog,
+.blog,
+.about,
+.faq {
+    margin-bottom: 80px;
+}
+
+/*Blog page */
+
+.blog-intro {
+    max-width: 100%;
+    min-height: 375px;
+    margin: 0 auto;
+    background-color: #14131C;
+    background-image: url(images//bg-4.webp), url(images//bg-3.webp);      
+    background-size: contain, cover;      
+    background-repeat: no-repeat, no-repeat;
+    background-position: 0 242px, center;
```

Comment:
background-position for first bg should be changed on bottom (or 100%)
`background-position: 0 bottom, center;`
to avoid this situation:
<img width="1728" alt="Screenshot 2024-12-04 at 20 32 55" src="https://github.com/user-attachments/assets/a8bbc1ef-b0c0-4896-805f-b58e735cf7e8">

Reply:
done